protocols/3db190/3db190.ot2.apiv2.py

```diff
@@ -0,0 +1,65 @@
+metadata = {
+    'protocolName': 'PCR/qPCR prep',
+    'author': 'Sakib <sakib.hossain@opentrons.com>',
+    'description': 'Custom Protocol Request',
+    'apiLevel': '2.7'
+}
+
+
+def run(protocol):
+
+    [total_samples, p20_mount, p1000_mount] = get_values(  # noqa: F821
+        "total_samples", "p20_mount", "p1000_mount")
+
+    total_samples = int(total_samples)
+
+    # Load Tip Racks
+    tipracks_20ul = [protocol.load_labware('opentrons_96_filtertiprack_20ul',
+                                           slot) for slot in ['7', '8']]
+    tipracks_1000ul = [protocol.load_labware(
+        'opentrons_96_filtertiprack_1000ul', slot) for slot in ['4', '5']]
+
+    # Load Plates
+    # 1 mL Deepwell Microtiter Plate
+    deepwell_plate = protocol.load_labware('eppendorf_96_deepwell_1000ul', 2)
+    # PCR Plate resting on top of the thermal block
+    pcr_plate = protocol.load_labware('enduraplate_96_wellplate_200ul', 3)
+
+    # Load Instruments
+    P20_single = protocol.load_instrument('p20_single_gen2', p20_mount,
```

Comment:
just a style note to go with `p20` and `p1000` rather than `P20_single` and `p1000_single`